prompt_toolkit/shortcuts/progress_bar/base.py

```diff
@@ -306,6 +306,7 @@ def __init__(self, progress_bar: ProgressBar,
                  total: Optional[int] = None) -> None:
 
         self.start_time = datetime.datetime.now()
+        self.stop_time = None
```

Comment:
I think this requires a type annotation, otherwise, Mypy will not allow assigning anything else but ``None`` to ``stop_time``.
```python
self.stop_time: Optional[datetime.datetime] = None
```